cpu/atmega_common/cpu.c

```diff
@@ -99,6 +108,7 @@ void cpu_init(void)
     periph_init();
 }
 
+#if defined (CPU_ATMEGA256RFR2)
```

Comment:
I think the other platforms could benefit from parts of ```BADISR_vect```.

```IRQ_STATUS```, ```SCIRQS```, ```BATMON``` need to be specific to ```CPU_ATMEGA256RFR2``` though.

Reply:
Then what is left for other platforms?